forge-game/src/main/java/forge/game/cost/CostAdjustment.java

```diff
@@ -485,6 +485,10 @@ private static boolean checkRequirement(final SpellAbility sa, final StaticAbili
                 if (!sa.isActivatedAbility() || sa.isManaAbility() || sa.isReplacementAbility()) {
                     return false;
                 }
+            } else if (type.equals("ManaAbility")) {
```

Comment:
This extension shouldn't be needed
Just try `ValidSpell$ Activated.ManaAbility` on the card instead

Reply:
You should not have dropped the ValidCard too